README.md

```diff
@@ -9,3 +9,79 @@ Based on https://github.com/exasol/error-reporting-java
 
 Major difference is this library is simplified by the string interpolation feature available in the C# language.
 
+## Installation
+
+- Make sure you have the .NET core SDK (use the dotnet SDK installation step on GitHub runners).
+
+- You'll need to add the Exasol Github NuGet repository to the NuGet package manager on your local machine or the GitHub CI runner, like so: 
+   - On GitHub CI runners:
+	```dotnet nuget add source --username <username/or ci user> --password ${{ secrets.GITHUB_TOKEN }} --store-password-in-clear-text --name github "https://nuget.pkg.github.com/EXASOL/index.json"```
+   - You can use a GitHub PAT on your local system:
+    ```dotnet nuget add source --username <username/or ci user> --password <yourPAT> --store-password-in-clear-text --name github "https://nuget.pkg.github.com/EXASOL/index.json"```
```

Comment:
```suggestion
   - You can use a GitHub PAT on your local system:
    ```
    dotnet nuget add source --username <username/or ci user> --password <yourPAT> --store-password-in-clear-text --name github "https://nuget.pkg.github.com/EXASOL/index.json"
    ```
```

Reply:
What do you think?